packages/guides/src/Renderer/InterlinkObjectsRenderer.php

```diff
@@ -19,8 +19,11 @@
 use phpDocumentor\Guides\Renderer\UrlGenerator\UrlGeneratorInterface;
 
 use function json_encode;
+use function sprintf;
+use function zlib_encode;
```

Comment:
We'll have to see if that zlib module is already configured for the docker-container and the GH Action build?

Reply:
Yes. that would be needed